apps/license-api/src/app/modules/license/license.types.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Pass,
+  PassDataInput,
+  Result,
+  RevokePassData,
+  VerifyPassData,
+} from '@island.is/clients/smartsolutions'
+
+export enum LicenseUpdateType {
+  PUSH = 'push',
+  PULL = 'pull',
+}
+
+export enum LicenseId {
+  FIREARM_LICENSE = 'FirearmLicense',
+  DISABILITY_LICENSE = 'DisabilityLicense',
+}
```

Comment:
I would use `LicenseType` instead of `LicenseId` as the parameter licenseId can easily be confused with a guid of some license.

According to my suggestions on the controller, I would change the value to be:
```suggestion
export enum LicenseId {
  FIREARM_LICENSE = 'firearm',
  DISABILITY_LICENSE = 'disability',
}
```
then we can actually provide these via the path be creating nice resource like url.